src/tests/test_magic.py

```diff
@@ -1987,3 +1987,100 @@ def test_accessing_previously_nonexisting_file(ip_empty, tmp_empty, capsys):
     ip_empty.run_cell("%sql SELECT * FROM 'data.csv' LIMIT 3")
     out, _ = capsys.readouterr()
     assert expected in out
+
+
+def test_warn_when_using_snippets_in_non_SELECT_command(ip_empty, capsys):
```

Comment:
looks like you're testing many things at once, use pytest's `parametrize` instead

Reply:
[Fixed](https://github.com/ploomber/jupysql/pull/878/files#diff-d327143929530bcece2cae65d59465708cdf87f499fd83b4bd939c332150f713R1992-R2036)